store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala

```diff
@@ -475,7 +475,7 @@ class PGSecondary(val config: Config) extends Secondary[SoQLType, SoQLValue] wit
     // re-create rollup metadata
     for { rollup <- rollups } RollupCreatedOrUpdatedHandler(pgu, postUpdateTruthCopyInfo, rollup)
     // re-create rollup tables
-    updateRollups(pgu, postUpdateTruthCopyInfo, dropFirst = false)
```

Comment:
Would be good to a note as to why this is important, since you did mention in slack it was time consuming to get to this conclusion.